.golangci.yml

```diff
@@ -5,6 +5,10 @@ run:
 issues:
   max-per-linter: 0
   max-same-issues: 0
+  exclude-rules:
+    - linters:
+        - stylecheck
+      text: "ST1003:"
```

Comment:
why?

Reply:
It validates that package names do not contain underscores, but `tfplugingen-framework` generates a package name with `resource_`/`datasource_` prefix and I can't control it.